src/components/Img/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, BoxProps } from '@mui/material';
+import { SyntheticEvent, useState } from 'react';
+
+/**
+ * This component will display an image given an src.
+ * The `fallback` prop has been added to make it easier to update the image in case the primary src fails.
+ */
+export const Img = ({
+	src: srcProp,
+	fallback,
+	...props
+}: BoxProps<'img'> & { fallback?: BoxProps<'img'>['src'] }) => {
```

Comment:
I would move in a interface for consistency and to not have to open another PR the day we will need img props